docs/tools_user_guide.md

```diff
@@ -341,13 +346,15 @@ Additionally, you can leverage labels (see [Labels](#labels)) to filter which mo
 
 > _Note_: Using bash regular expressions and filtering model by hashes are mutually exclusive. To filter models by hashes, provide the full path of the Python script rather than a regular expression.
 
+> _Note_: ONNX file input currently supports only models of size less than 2 GB. ONNX files passed directly into `benchit *.onnx` are benchmarked as-is without applying any additional build stages.
```

Comment:
Do we benchmark as is? We still do the ONNXruntime Graph optimization and FP16 conversion right?

Reply:
Yep it's as-is. See the command line output in the description. No optimize or convert stages - it's just BYO ONNX. We can add more options later if people want, however this feature is currently intended just to support a pre-made LLaMA ONNX file.